nservicebus/messaging/discard-old-messages_msmq_core_[6.0,).md

```diff
@@ -0,0 +1,3 @@
+MSMQ continuously checks the TimeToBeReceived attribute of all queued messages. As soon as the message has expired, it is removed from the queue, and disk space reclaimed. MSMQ will however only allow a single TimeToBeReceived for all messages in a transaction. It will silently copy the TimeToBeReceived from the first message enlisted to all other messages in the same transaction, leading to a potential message loss scenario. This issue, unfortunately, make it impossible to support setting TimeToBeReceived for messages sent from a transactional MSMQ endpoint.
```

Comment:
👍 for the note to be in partial but the first paragraph is the same for both so I think should stay in the main doco for now. No reason for duplicating it just now.